### DISCUSSION TITLE
Integração com GoogleSheets

### DISCUSSION THREAD

**Discussion by gnosouza** [DefectDojo/django-DefectDojo#7340] [General]
Alguma maneira de gerar e sincronizar todos os produtos ao invés de selecionar produto a produto no GoogleSheets?

**Comment by valentijnscholten** [comment]
The sheet can be generated on a per product basis only.

**Reply by mtesauro** [reply]
The only way I can think to do that is to call the REST API and create a CSV/spreadsheet of all products yourself.

I did do that years ago at a place I worked.
